packages/runtime-handler/__tests__/dev-runtime/utils/requireFromProject.test.ts

```diff
@@ -26,12 +25,13 @@ jest.mock(
 );
 
 describe('requireFromProject', () => {
-  test('should return project based by default', () => {
-    const mod = requireFromProject(PROJECT_DIR, '@twilio/test-dep');
-    expect(mod['__TYPE__']).toBe('PROJECT_BASED');
-    const mod2 = require('@twilio/test-dep');
-    expect(mod2['__TYPE__']).toBe('BUILT_IN');
-  });
+  // npm hoists the package to root node_modules making this test not possible currently
```

Comment:
Do we need this comments here?

Reply:
a reminder to uncomment it when we migrate off npm